.github/workflows/update-base-image.yml

```diff
@@ -33,6 +33,11 @@ jobs:
       - 
         name: Checkout repository
         uses: actions/checkout@v4
+      - name: Get package name
+        id: cargo-get
+        uses: nicolaiunrein/cargo-get@master
+        with:
+          subcommand: package.version
```

Comment:
Isn't it easier and more reliable to get the latest version by using `github.ref` which to my knowledge gets the most recent tag?

Reply:
Easier, yes, but that would mean the ref wouldn't change until you release a tag, even if the underlying image is running something more recent than the latest tag.

Totally up to you here, this is my preference but it's your repo, happy to adjust to whatever you prefer!

Reply:
Nevermind, github.ref_name point to the ref that triggered the job, not the latest  tag.
This job is set to go for every master push so what I said does not make any sense.